test/functional/rpc_misc.py

```diff
@@ -55,13 +55,27 @@ def run_test(self):
 
         assert_raises_rpc_error(-8, "unknown mode foobar", node.getmemoryinfo, mode="foobar")
 
-        self.log.info("test logging")
+        self.log.info("test logging rpc and help")
+
+        # Test logging RPC returns the expected number of logging categories.
+        assert_equal(len(node.logging()), 34)
```

Comment:
```suggestion
        assert_equal(len(node.logging()), 36)
```
because of https://github.com/dashpay/dash/pull/5707

Reply:
should fix https://github.com/dashpay/dash/pull/5663#issuecomment-1816135745